DataFormats/L1THGCal/interface/HGCalClusterT.h

```diff
@@ -148,8 +150,6 @@ namespace l1t
       }
 
       uint32_t subdetId() const {return detId_.subdetId();} 
-      uint32_t layer() const {return detId_.layer();}
```

Comment:
@jbsauvan 
What happens to this accessor and the line below ?

Reply:
`detId_` needs to be generic (`DetId`) in order to be able to deal with the different detid schemes. The `layer` should now be retrieved through the `HGCalTriggerTools`:
https://github.com/PFCal-dev/cmssw/blob/hgc-tpg-devel-CMSSW_10_5_0_pre1/L1Trigger/L1THGCal/interface/HGCalTriggerTools.h#L46